### PR TITLE
new Margin API contract feature added

### PR DESCRIPTION
**Pull request by Arvind-Acc**
New Margin API contract feature added in smartConnect file

### CODE REVIEW

**Review by anurag-amx — CHANGES_REQUESTED** [review]
as requested to use common method.

**Comment by Arvind-Acc** [comment]
created common method

File: SmartApi/smartConnect.py [hunk 1]
```diff
@@ -433,6 +436,30 @@ def searchScrip(self, exchange, searchscrip):
...
+        if access_token:
+            headers["Authorization"] = "Bearer " + access_token
+        response = requests.get(url, headers=headers)
+        if response.status_code == 200:
+            data = json.loads(response.text)
+            return data
+        else:
+            raise Exception(f"Error: {response.status_code}, {response.text}")
+        
+    def ind_order_details(self, qParam):
```
Comment (anurag-amx) [inline SmartApi/smartConnect.py:450]: make method name - individual_order_detail

**Review by anurag-amx — CHANGES_REQUESTED** [review]

File: SmartApi/smartConnect.py (hunk 1 above)
Comment (Arvind-Acc) [reply SmartApi/smartConnect.py:450]: rename the function name ind_order_detail to individual_order_detail

**Pull request closed without merging**